### DISCUSSION TITLE
Using words to "guide" the transcription.

### DISCUSSION THREAD

**Discussion by LucasAssis00** [openai/whisper#1924] [Q&A]
Hello,
I am trying to use Whisper to identify pronounced items from a list. It has a few thousand of descriptions, some of them are a bit area specific, so I tryed using "initial_prompt" to guide the transcription. I tryed using a list but it was not accepted, so separated a few words with commas. It worked when I tested with a music but in the actual project it did not work. Is there a more precise way of doing this?

**Comment by LucasAssis00** [comment]
![image](https://github.com/openai/whisper/assets/92377740/84c90e26-53db-474d-a67b-34beae1405be)


![Captura de tela de 2023-12-28 10-51-56](https://github.com/openai/whisper/assets/92377740/51125d39-0b9b-43d6-ba50-d872bcb72eff)

**Comment by ryanheise** [comment]
The prompt can be thought of as a transcript of the audio that could plausibly have appeared just prior to the actual start of the audio (i.e. if you imagine there being an extra 30 seconds of audio at the start, what might the speaker have plausibly said?) A list of words is not going to be a plausible scenario, so it's unlikely to give good results.

**Comment by glangford** [comment]
See also the answer in this discussion:
* https://github.com/openai/whisper/discussions/963
